src/jobflow_remote/jobs/runner.py

```diff
@@ -1073,9 +1073,9 @@ def update_batch_jobs(self) -> None:
                 )
 
                 if submit_result.status == SubmissionStatus.FAILED:
-                    logger.error(f"submission failed. {submit_result!r}")
+                    logger.exception(f"submission failed. {submit_result!r}")
```

Comment:
I don't think this change is valid; my understanding is that logger.exception needs to be inside an exception handler but nothing is being raised here (there will be many such cases)

Reply:
thanks, i'll look into this for the 2nd PR